array-loops.js

```diff
@@ -3,7 +3,11 @@
 // --------------------------
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+for(let i = 0; i < names.length; i++){
```

Comment:
Voor de leesbaarheid van je code zou ik tussen je array en je for-loop een witregel doen.
Bij deze regel kun je beter na het laatste haakje en de accolade een spatie doen.